Bottom: 2a220ba48ee696047d08a38fab6dd85f8fc7f7d9
Top:    d8fd916f4c7af59f39bbac80fff5e950dc0b2682
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2008-11-04 09:46:07 -0500

Refresh of jds-lirc-mce2

---

diff --git a/drivers/input/ir/Makefile b/drivers/input/ir/Makefile
index ab0da3f..0bdafb0 100644
--- a/drivers/input/ir/Makefile
+++ b/drivers/input/ir/Makefile
@@ -8,4 +8,5 @@ ir-objs := ir-core.o ir-configfs.o
 
 
 obj-$(CONFIG_IR_GPT)		+= ir-gpt.o
+obj-$(CONFIG_IR_MCEUSB2)	+= ir-mceusb2.o
